CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
 * [ENHANCEMENT] Azure client: Improve memory usage for large object storage downloads. #2408
 * [ENHANCEMENT] Distributor: Add `-distributor.instance-limits.max-inflight-push-requests-bytes`. This limit protects the distributor against multiple large requests that together may cause an OOM, but are only a few, so do not trigger the `max-inflight-push-requests` limit. #2413
 * [ENHANCEMENT] Distributor: Drop exemplars in distributor for tenants where exemplars are disabled. #2504
+* [ENHANCEMENT] Runtime Config: Add dskit functionality to specify multiple comma-separated yaml files in `runtime-config.file` that will be merged in left to right order. #2583
```

Comment:
[nit] Please prefix CLI flags with `-`. The final user is not aware of what dskit. No need to mention it here.

```suggestion
* [ENHANCEMENT] Runtime Config: Allow to specify multiple comma-separated yaml files in `-runtime-config.file` that will be merged in left to right order. #2583
```